src/main/java/uk/gov/hmcts/reform/professionalapi/controller/SuperController.java

```diff
@@ -174,17 +178,17 @@
                 return ResponseEntity.status(responseEntity.getStatusCode()).body(responseEntity.getBody());
             }
         }
-        OrganisationResponse organisationResponse = organisationService.updateOrganisation(organisationCreationRequest, organisationIdentifier);
+        OrganisationResponse organisationResponse = organisationService.updateOrganisation(organisationCreationRequest, orgId);
         return ResponseEntity.status(200).build();
     }
 
     private ResponseEntity createUserProfileFor(ProfessionalUser professionalUser, List<String> roles, boolean isAdminUser) {
         log.info("Creating user...");
         List<String> userRoles = isAdminUser ? prdEnumService.getPrdEnumByEnumType(prdEnumRoleType) : roles;
         UserProfileCreationRequest userCreationRequest = new UserProfileCreationRequest(
-                professionalUser.getEmailAddress(),
-                professionalUser.getFirstName(),
-                professionalUser.getLastName(),
+                PbaAccountUtil.removeAllSpaces(professionalUser.getEmailAddress()),
```

Comment:
Remove whitespaces at inviteUserToOrganisaation method and not here, because this will become inconsistent when ProfessionalUser is saved without removing whitespaces.